### DISCUSSION TITLE
Klipper Screen won't compile on FLSUN Speeder Pad

### DISCUSSION THREAD

**Discussion by cottec** [KlipperScreen/KlipperScreen#778] [General]
Hey guys,
maybe one of you has some good ideas what i can try to get Klipper Screen running on my machine?
This fork is working for everyone except for me...
https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad

But i dont even get the original from here compiled, errors and logs are discussed in this issue:
https://github.com/jordanruthe/KlipperScreen/issues/776#issuecomment-1304317883

Thanks for your help :)

**Comment by alfrix** [comment]
resolved a long time ago
